src/interface/events.ts

```diff
@@ -17,6 +17,9 @@ export enum InterfaceEventName {
   NAVBAR_SEARCH_EXITED = 'Navbar Search Exited',
   NAVBAR_SEARCH_SELECTED = 'Navbar Search Selected',
   RISK_CHECKED = 'Risk Checked',
+  SEND_FIAT_TOGGLED = 'Send Fiat Toggled',
```

Comment:
What does this represent? Can this be an element clicked event?

Reply:
yeah do we not need to put clicked events in here?